### PR TITLE
RuboCop

### PR DESCRIPTION
**Pull request by bquorning**
Should we enable some style checks on this repository?

cc/ @jacobat @grosser

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -34,10 +34,18 @@ if defined?(RSpec)
   task :default => "spec"
 end
 
+require 'rubocop/rake_task'
+desc 'Run RuboCop on the lib directory'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  # task.patterns = %w(Rakefile lib/**/*.rb spec/**/*.rb)
+  task.formatters = %w(progress)
+  task.fail_on_error = true
```
Comment (grosser) [inline Rakefile]: that's not the default ?

File: .travis.yml [hunk 2]
```diff
@@ -8,4 +8,8 @@ rvm:
   - 2.3.0
   - jruby
 
+script:
```
Comment (grosser) [inline .travis.yml]: if it's part of travis it should be part of rake too ... something like https://github.com/abonas/kubeclient/pull/166

File: Rakefile [hunk 3]
```diff
@@ -34,10 +34,18 @@ if defined?(RSpec)
   task :default => "spec"
 end
 
+require 'rubocop/rake_task'
```
Comment (grosser) [inline Rakefile]: require to the top ?

**Comment by grosser** [comment]
👍 ideally resolve more of the warnings, but that can be done in followup PRs ...

File: Rakefile (hunk 3 above)
Comment (bquorning) [inline Rakefile]: Sure thing

File: .travis.yml (hunk 2 above)
Comment (bquorning) [inline .travis.yml]: Except the rspec tasks are inside a conditional. Don’t wanna mess with that right now.

File: Rakefile (hunk 1 above)
Comment (bquorning) [inline Rakefile]: It sure is. Thanks for noticing.

**Comment by bquorning** [comment]
> ideally resolve more of the warnings, but that can be done in followup PRs ...

Yes, more will follow.

**Pull request merged**